joplin/base/migrations/0084_remove_topicpage_external_services.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.4 on 2019-09-05 17:03
```

Comment:
The migrations here are confusing to me,

It seems as though the most recent migration should be this, as this is the code change that we're making.

Is there something strange about how merging migrations is working?

Reply:
Its working as intended. Having all the migrations there would be more important if we had other branches depending on them. Since our branch structure is pretty simple and joplin is only being used by the alpha project atm it's probably a bit overkill.